src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -105,9 +105,29 @@
              :else {:success typed})))
    :render
    (fn [{:keys [props state after-update]}]
-     (let [{:keys [editing?]} props]
+     (let [{:keys [editing? writer?]} props]
        [:div {}
```

Comment:
We want the import attributes button to only be visible to those who have write access, which is why we're passing that in